packages/core/src/components/CopyTextButton/CopyTextButton.js

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { Fragment } from 'react';
+import PropTypes from 'prop-types';
+import { IconButton, Tooltip, withStyles } from '@material-ui/core';
+import CopyIcon from '@material-ui/icons/FileCopy';
+
+const buttonStyles = theme => ({
+  button: {
+    '&:hover': {
+      backgroundColor: theme.palette.highlight,
+      cursor: 'pointer',
+    },
+  },
+});
+
+/**
+ * Copy text button with visual feedback in the form of
+ *  - a hover color
+ *  - click ripple
+ *  - Tooltip shown when user has clicked
+ *
+ *  Properties:
+ *  - text: the text to be copied
+ *  - tooltipDelay: Number os ms to show the tooltip, default: 1000ms
+ *  - tooltipText: Text to show in the tooltip when user has clicked the button, default: "Text
+ * copied to clipboard"
+ *
+ * Example:
+ *    <CopyTextButton text="My text that I want to be copied to the clipboard" />
+ */
+class CopyTextButton extends React.Component {
+  static propTypes = {
+    text: PropTypes.string.isRequired,
+    tooltipDelay: PropTypes.number,
+    tooltipText: PropTypes.string,
+  };
+
+  static defaultProps = {
+    tooltipDelay: 1000,
+    tooltipText: 'Text copied to clipboard',
+  };
+
+  state = {
+    open: false,
+  };
+
+  handleTooltipClose = () => {
+    this.setState({ open: false });
+  };
+
+  handleTooltipOpen = () => {
+    this.setState({ open: true });
+  };
+
+  handleCopyClick = e => {
+    e.stopPropagation();
+    this.handleTooltipOpen();
+    try {
+      this.clipboardInput.select();
+      document.execCommand('copy');
+    } catch (error) {
+      // console.error(error);
```

Comment:
What should be done instead of logging? 🤔

Reply:
Use the error api to report it to the app :grin:

```
const errorApi = useApi(errorApiRef);
errorApi.post(error);
```

Reply:
Fixed! Thanks